cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -98,7 +98,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
-  name: rook-ceph-operator-psp-user
+  name: rook-ceph-system-psp-user
   labels:
     operator: rook
     storage-backend: ceph
```

Comment:
LGTM
Could also update/fix the docs https://rook.io/docs/rook/master/rbac.html, could you also fix the namespace names for the PSP stuff, e.g. `rook-system` must be replaced with `rook-ceph-system`

Reply:
good catch, i do see a couple places to update in the doc